src/component/comment/CommentDislikes.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import Comment, {CommentReaction} from "../../model/Comment";
+import User from "../../model/User";
+import {connect} from "react-redux";
+import TermItState from "../../model/TermItState";
+import {FaRegThumbsDown, FaThumbsDown} from "react-icons/fa";
+import withI18n, {HasI18n} from "../hoc/withI18n";
+import {injectIntl} from "react-intl";
+import {isMine} from "./CommentLikes";
+
+interface CommentDislikesProps extends HasI18n {
+    reactions: CommentReaction[];
+    comment: Comment;
+
+    currentUser: User;
+    addReaction: (comment: Comment, reactionType: string) => void;
```

Comment:
maybe some type for reactionTypes - like/dislike ?

Reply:
Added constant with literal values. Couldn't use enum, asi t does not allow computed values (NS + Like/Dislike)